client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -27,7 +47,7 @@ float intensity (vec3 color)
 
 float get_rgb_height (vec2 uv)
```

Comment:
Dammit, same thing here with the spaces.  You really like those spaces, don't you?